src/forms/FormFieldAddress/AddressDrawer/AddressDrawer.tsx

```diff
@@ -363,20 +362,20 @@ const AddressDrawer = (props: AddressDrawerProps): ReactElement => {
 					type: "text",
 					label: undefined,
 					size: "lg",
-				} as FieldDef<TextFieldDef>,
+				},

```

Comment:
For consistency please move the "as FieldDef[]"s next to their respective "const fields"